content/200-concepts/050-overview/100-what-is-prisma/index.mdx

```diff
@@ -20,7 +20,12 @@ Prisma is an [open source](https://github.com/prisma/prisma) next-generation ORM
 
 - **Prisma Client**: Auto-generated and type-safe query builder for Node.js & TypeScript
 - **Prisma Migrate**: Migration system
-- **Prisma Studio**: GUI to view and edit data in your database
+- **Prisma Studio**: GUI to view and edit data in your database.
+  <Admonition>
+
+  **Prisma Studio** is the only part of Prisma ORM that is not open source. You can only run Prisma Studio locally. At the same time, Prisma Studio is also integrated in our commercial offering [Prisma Data Platform](/data-platform/about) under the name [**Data Browser**](/data-platform/data-browser) where you can view and edit data for each project and grant other team members the permission to only view or also edit data based on their [role](/data-platform/members/roles-permissions).
```

Comment:
```suggestion
  **Prisma Studio** is the only part of Prisma ORM that is not open source. You can only run Prisma Studio locally. Prisma Studio is also integrated in our commercial offering [Prisma Data Platform](/data-platform/about) under the name [**Data Browser**](/data-platform/data-browser). In Data Browser, you can view and edit data for each project and grant other team members the permission to view or edit data based on their [role](/data-platform/members/roles-permissions).
```

I made a few suggestions to reduce the word count (while still preserving the sense). I also split the last sentence, because it was quite long - I think I've preserved the sense here - we are saying that Data Browser is the tool where you can view and edit data and grant permissions?